video/rest/compositions/compose-chess/compose-chess.5.x.cs

```diff
@@ -4,44 +4,56 @@
 using System.Collections.Generic;
 using Twilio;
 using Twilio.Rest.Video.V1;
-using static Twilio.Rest.Video.V1.CompositionResource;
+
 
 class Program
 {
     static void Main(string[] args)
     {
-        // Find your API Key SID and Secret at twilio.com/console
-        // To set up environmental variables, see http://twil.io/secure
-        const string apiKeySid = Environment.GetEnvironmentVariable("TWILIO_API_KEY_SID");
-        const string apiKeySecret = Environment.GetEnvironmentVariable("TWILIO_API_KEY_SECRET");
+        // Find your Account SID and Auth Token at twilio.com/console
+        // and set the environment variables. See http://twil.io/secure
+        string accountSid = Environment.GetEnvironmentVariable("TWILIO_ACCOUNT_SID");
+        string authToken = Environment.GetEnvironmentVariable("TWILIO_AUTH_TOKEN");
+
+        TwilioClient.Init(accountSid, authToken);
 
-        TwilioClient.Init(apiKeySid, apiKeySecret);
+        var audioSources = new List<string> {
+            "*"
+        };
 
-        var layout = new
+        var videoLayout = new Dictionary<string, Object>()
         {
-            chess_table = new
-            {
-              x_pos = 10,
-              y_pos = 0,
-              width = 1260,
-              height = 720,
-              max_rows = 3,
-              max_columns = 3,
-              reuse = "show_newest",
-              cells_excluded = new int[]{1,3,5,7},
-              video_sources = new string[]{"*"}
-            }
+            {"chess_table", new Dictionary<string, Object>()
+                {
+                    {"x_pos", 10},
+                    {"y_pos", 0},
+                    {"width", 1260},
+                    {"height", 720},
+                    {"max_rows", 3},
+                    {"max_columns", 3},
+                    {"reuse", "show_newest"},
+                    {"cells_excluded", new object [] {
```

Comment:
Hmm I wonder if this is just the way our dynamic code samples render these, because they might not know the answers here. (Although that being said, Java seems to understand when things are strings...). I can update this.